projects/ad469x_fmc/coraz7s/system_bd.tcl

```diff
@@ -0,0 +1,43 @@
+###############################################################################
+## Copyright (C) 2020-2024 Analog Devices, Inc. All rights reserved.
+### SPDX short identifier: ADIBSD
+###############################################################################
+
+source $ad_hdl_dir/projects/common/coraz7s/coraz7s_system_bd.tcl
+source $ad_hdl_dir/projects/scripts/adi_pd.tcl
+
+# specify the spi reference clock frequency in MHz
+set spi_clk_ref_frequency 160
+
+# specify ADC resolution -- supported resolutions 16 bits
+set adc_resolution 16
+
+# specify ADC sampling rate in samples/seconds
+set adc_sampling_rate 1000000
```

Comment:
remove useless variables

Reply:
Except the adc_resolution 16, the mentioned variables are used in ad469x_bd.tcl